<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>–ù–∞—Å—Ç—Ä–æ–π—Ç–µ –ü–∏—Ü—Ü—É –ü–æ–¥ –°–µ–±—è! (–í–∞—Ä—Ç–∞–Ω—è–Ω 4331)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="GroupBoxPizzaSize">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
     </font>
    </property>
    <property name="title">
     <string>–†–∞–∑–º–µ—Ä –ü–∏—Ü—Ü—ã</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>261</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="PizzaSize25Label">
          <property name="text">
           <string>25 —Å–º</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="PizzaSize30Label">
          <property name="text">
           <string>30 —Å–º</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="PizzaSize35Label">
          <property name="text">
           <string>35 —Å–º</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="PizzaSizeSlider">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>3</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="sliderPosition">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GroupBoxPizzaDough">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
     </font>
    </property>
    <property name="title">
     <string>–¢–µ—Å—Ç–æ</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>361</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QRadioButton" name="PizzaDoughTradRadioButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>–¢—Ä–∞–¥–∏—Ü–∏–æ–Ω–Ω–æ–µ</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="PizzaDoughThinRadioButton">
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="text">
         <string>–¢–æ–Ω–∫–æ–µ</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GroupBoxPizzaIngredients">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>621</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
     </font>
    </property>
    <property name="title">
     <string>–î–æ–±–∞–≤–∏—Ç—å –ø–æ –≤–∫—É—Å—É :)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>601</width>
       <height>210</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="PizzaIngredientsCheesesLabel">
          <property name="text">
           <string>   –°—ã—Ä—ã</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="PizzaMozzarellaCheckBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>–ú–æ—Ü–∞—Ä–µ–ª–ª–∞</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="PizzaCheddarCheckBox">
            <property name="text">
             <string>–ß–µ–¥–¥–µ—Ä</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="PizzaParmesanCheckBox">
            <property name="text">
             <string>–ü–∞—Ä–º–µ–∑–∞–Ω</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="PizzaGorgonzolaCheckBox">
            <property name="text">
             <string>–ì–æ—Ä–≥–æ–Ω–∑–æ–ª–∞</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="PizzaBrinzaCheckBox">
            <property name="text">
             <string>–ë—Ä—ã–Ω–∑–∞</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="PizzaCheeseSideCheckBox">
            <property name="text">
             <string>–°—ã—Ä–Ω—ã–π –ë–æ—Ä—Ç–∏–∫</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="PizzaIngredientsMeatLabel">
          <property name="text">
           <string>   –ú—è—Å–æ</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="PizzaChickenCheckBox">
            <property name="text">
             <string>–¶—ã–ø–ª–µ–Ω–æ–∫</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="PizzaPepperoniCheckBox">
            <property name="text">
             <string>–ü–µ–ø–ø–µ—Ä–æ–Ω–∏</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="PizzaBaconCheckBox">
            <property name="text">
             <string>–ë–µ–∫–æ–Ω</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="PizzaHamCheckBox">
            <property name="text">
             <string>–í–µ—Ç—á–∏–Ω–∞</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="PizzaChorizoCheckBox">
            <property name="text">
             <string>–ß–æ—Ä–∏–∑–æ</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="PizzaMeatballsCheckBox">
            <property name="text">
             <string>–§—Ä–∏–∫–∞–¥–µ–ª—å–∫–∏</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="PizzaIngredientsVegiesLabel">
          <property name="text">
           <string>   –û–≤–æ—â–∏ –∏ –î—Ä—É–≥–æ–µ</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="PizzaJalapenoCheckBox">
            <property name="text">
             <string>–•–∞–ª–∞–ø–µ–Ω—å–æ</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="PizzaChampignogsCheckBox">
            <property name="text">
             <string>–®–∞–º–ø–∏–Ω—å–æ–Ω—ã</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="PizzaCucumbersCheckBox">
            <property name="text">
             <string>–û–≥—É—Ä—á–∏–∫–∏</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="PizzaRedOnionCheckBox">
            <property name="text">
             <string>–ö—Ä–∞—Å–Ω—ã–π –õ—É–∫</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="PizzaSweerPepperCheckBox">
            <property name="text">
             <string>–°–ª–∞–¥–∫–∏–π –ü–µ—Ä–µ—Ü</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="PizzaTomatoesCheckBox">
            <property name="text">
             <string>–¢–æ–º–∞—Ç—ã</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLabel" name="PizzaIngredientsFruitsLabel">
          <property name="text">
           <string>   –§—Ä—É–∫—Ç—ã</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QCheckBox" name="PizzaPineappleCheckBox">
            <property name="text">
             <string>–ê–Ω–∞–Ω–∞—Å</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="PizzaPearCheckBox">
            <property name="text">
             <string>–ì—Ä—É—à–∞</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="PizzaMandarinCheckBox">
            <property name="text">
             <string>–ú–∞–Ω–¥–∞—Ä–∏–Ω</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="PizzaBananasCheckBox">
            <property name="text">
             <string>–ë–∞–Ω–∞–Ω</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="PizzaAppleCheckBox">
            <property name="text">
             <string>–Ø–±–ª–æ–∫–æ</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="PizzaKiwiCheckBox">
            <property name="text">
             <string>–ö–∏–≤–∏</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>601</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="PizzaChooseAllPushButton">
        <property name="text">
         <string>–í—ã–±—Ä–∞—Ç—å –í—Å–µ –ò–Ω–≥—Ä–µ–¥–∏–µ–Ω—Ç—ã 0_0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PizzaUnChooseAllPushButton">
        <property name="text">
         <string>–°–Ω—è—Ç—å –í—Å–µ –í—ã–±–æ—Ä—ã</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
     </font>
    </property>
    <property name="text">
     <string>–í–∞—à–∞ –ò–¥–µ–∞–ª—å–Ω–∞—è –ü–∏—Ü—Ü–∞ üéî</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>480</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QLabel" name="PizzaPromoLabel">
         <property name="font">
          <font>
           <family>Century Gothic</family>
          </font>
         </property>
         <property name="text">
          <string>–ü—Ä–æ–º–æ–∫–æ–¥</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="PizzaPromocode">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="PromoFeedback">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>191</width>
      <height>131</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>189</width>
       <height>129</height>
      </rect>
     </property>
     <widget class="QScrollBar" name="verticalScrollBar">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>0</y>
        <width>17</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>6</pointsize>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="PizzaHintPromoLabel1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>560</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="text">
     <string>–ü–æ—Ö–æ–∂–µ, —ç—Ç–æ –≤–∞—à –ø–µ—Ä–≤—ã–π –∑–∞–∫–∞–∑... –í–≤–µ–¥–∏—Ç–µ –ø—Ä–æ–º–æ–∫–æ–¥ FIRST –∏ –ø–æ–ª—É—á–∏—Ç–µ —Å–∫–∏–¥–∫—É -25% –Ω–∞ –≤—Å—ë!</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="CheckSumm">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>480</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
